### DISCUSSION TITLE
How do I enable sleep again?

### DISCUSSION THREAD

**Discussion by risbi0** [simeononsecurity/Windows-Optimize-Harden-Debloat#75] [Q&A] [answered]
I installed the script using the automatic method. The sleep feature is now gone but is there a way to re-enable it?

**Comment by simeononsecurity** [comment, marked as answer]
There are many reasons why you should disable sleep. The top one being that if your system isn't on, it isn't logging. This is a concern for the DoD and NIST. Based upon the STIGs documented in the readme, this is an option we don't agree with removing altogether. However we acknowledge that it isn't a desired configuration for users using a laptop.

There are two ways that sleep gets disabled in this script:
1. **The power plan:** In this script, sleep mode is often disabled by modifying the power plan settings. To re-enable sleep in Windows 10 through the power plan:
   - Click on the Windows Start button and select "Control Panel."
   - In the Control Panel, navigate to "Power Options."
   - In the Power Options window, select "Change plan settings" for the active power plan.
   - Click on "Change advanced power settings."
   - Look for the "Sleep" option in the list of settings and expand it.
   - Adjust the sleep settings to your preference, typically by setting the "Sleep after" time to a non-zero value.
   - Click "Apply" and then "OK" to save the changes.

2. **Group Policy Objects (GPO):** To re-enable sleep using GPO in Windows 10:
   - Press Win + R to open the Run dialog.
   - Type "gpedit.msc" and press Enter to open the Group Policy Editor.
   - Navigate to "Computer Configuration" > "Administrative Templates" > "System" > "Power Management" > "Sleep Settings."
   - In the right pane, double-click on "Allow standby states (S1-S3) when sleeping (plugged in)."
   - Select "Enabled" to allow sleep mode when plugged in.
   - Also, double-click on "Allow standby states (S1-S3) when sleeping (on battery)."
   - Select "Enabled" to allow sleep mode when on battery.
   - Click "Apply" and then "OK" to save the changes.
